### DISCUSSION TITLE
unsubscribe option in diary study

### DISCUSSION THREAD

**Discussion by fidadoma** [rubenarslan/formr.org#511] [Q&A] [answered]
Hi everyone,
thanks, Ruben, for moving the group here. I have a question that is impossible to look at on the forum (given the fact that I need to find the word _unsubscribe_). We are planning a diary study in which participants would receive emails for 14 days, and we want to give them the option to unsubscribe. We could do that in the survey; however, is there an option how to pass them a link in the email reminder that would move them to a different part in the run? Or only option is to use the checkbox in the diary survey?
All the best
F

**Comment by rubenarslan** [comment, marked as answer]
hi @fidadoma, yes this is poorly documented, if at all. There is a page for each run at `/settings`, so e.g. [widgets.formr.org/settings](https://widgets.formr.org/settings) where people can unsubscribe from emails. This does not move them anywhere in the run. They will simply get stuck on the next email in the run with the note "email_skipped_user_disabled" for you in the user overview and a note to them that they disabled emails if they access the run. You can then move them elsewhere in the run.
If you have ideas how this could be better designed, I'm all ears. We should also document this more and maybe include a link to this page in emails by default.

**Reply by fidadoma** [reply]
Wow, I missed that option, that is very handy, and this solves our problem. I think that this design is fine; I was just not able to find it in the documentation. Thanks!

**Reply by dfeistauer** [reply]
Hi @fidadoma,
I did something like this with a get-Parameter to push participants to a special position in a run.

E-Mail-Modul
![image](https://github.com/rubenarslan/formr.org/assets/44641119/8400483c-737c-4b00-8943-32ac043fcea5)

You need a survey. Participants don't see this survey.
![image](https://github.com/rubenarslan/formr.org/assets/44641119/05f08f88-d62e-4203-8508-ff36fcb87ce7)

use the data for a jump
![image](https://github.com/rubenarslan/formr.org/assets/44641119/bf74209d-06da-428a-8bc4-6211f1ea8751)

Final position in my run.
![image](https://github.com/rubenarslan/formr.org/assets/44641119/b800e696-facb-496b-97da-6064ee4a0619)

**Reply by fidadoma** [reply]
Thank you very much, this is very helpful!
